plugins/lookup/bitwarden.py

```diff
@@ -109,10 +114,19 @@ def get_field(self, field, search_value, search_field="name"):
         """
         matches = self._get_matches(search_value, search_field)
 
-        if field:
+        if field in ['autofillOnPageLoad', 'password', 'passwordRevisionDate', 'totp', 'uris', 'username']:
             return [match['login'][field] for match in matches]
-
-        return matches
+        elif not field:
+            return matches
+        else:
+            custom_field_matches = []
+            for match in matches:
+                for custom_field in match['fields']:
+                    if custom_field['name'] == field:
+                        custom_field_matches.append(custom_field['value'])
+            if not len(custom_field_matches):
```

Comment:
One last comment:
```suggestion
            if not custom_field_matches:
```
On the condition: shouldn't it only error out if matches were found, but the field wasn't found in there? I.e.
```suggestion
            if matches and not custom_field_matches:
```
?